src/Components/benchmarkapps/BlazingPizza.Server/NuGet.config

```diff
@@ -1,7 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
+<!-- Auto generated file from Gardener Plugin CentralFeedServiceAdoptionPlugin -->
 <configuration>
     <packageSources>
         <clear />
-        <add key="NuGet.org" value="https://api.nuget.org/v3/index.json" />
+        <add key="internal_PublicPackages" value="https://pkgs.dev.azure.com/dnceng/internal/_packaging/internal_PublicPackages/nuget/v3/index.json" />
```

Comment:
Will this feed be accessible in public builds?

Reply:
I don't know but hope so

Reply:
In any case, I was able to `yarn install` things locally in this branch

Reply:
Doubt, isn't this why SignalR/SignalR is failing for the similar change?

Reply:
It's possible this was all for naught. I'm checking offline w/ @magleaso

Reply:
OIC: We used <https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-public-npm/npm/registry/> in [jquery-validation-unobtrusive](https://github.com/aspnet/jquery-validation-unobtrusive). I'll switch feeds here then fix up AspLabs and SignalR.

Reply:
done here